engine/function/src/templates/Numeric.ftl

```diff
@@ -1280,52 +1292,64 @@ public class Numeric {
         if (isNaN(sorted[sorted.length - 1])) {
             return Double.NaN; // Any NaN will pollute the result and NaN always sorted to the end.
         }
-        </#if>
 
         int nullStart = -1;
         int nullCount = 0;
-
         for (int i = 0; i < n; i++) {
             final ${pt.primitive} val = sorted[i];
             if (val > ${pt.null}) {
                 break; // no more NULL possible
-            } else if (nullStart == -1) {
-                if (isNull(val)) {
+            }
+            if (isNull(val)) {
+                nullCount++;
+                if (nullStart == -1) {
                     nullStart = i;
-                    nullCount++;
                 }
-            } else if (isNull(val)) {
-                nullCount++;
             }
         }
+        <#else>
+        int nullCount = 0;
+        for (int i = 0; i < n && isNull(sorted[i]); i++) {
+            nullCount++;
+            <#if pt.valueType.isFloat >
+            if (nullStart == -1) {
+                nullStart = i;
+            }
+            </#if>
```

Comment:
I'm not convinced this code is reachable.

<img width="832" alt="image" src="https://github.com/deephaven/deephaven-core/assets/5250374/38310ed9-beb9-4da4-80ff-c97de6e9c1c8">